### PR TITLE
feat(coil-privacypass-sjcl): update types

### PR DESCRIPTION
**Pull request by sublimator**

### CODE REVIEW

File: packages/coil-privacypass-sjcl/src/issuance.ts [hunk 1]
```diff
@@ -15,25 +21,31 @@ export const CACHED_COMMITMENTS_STRING = 'cached-commitments'
...
+  if (isRawIssueResponse(issueResp)) {
     signatures = issueResp.sigs
     batchProof = issueResp.proof
     version = issueResp.version
+    prng = issueResp.prng
   } else {
     batchProof = issueResp[issueResp.length - 1]
     signatures = issueResp.slice(0, issueResp.length - 1)
+    // TODO
+    version = (undefined as unknown) as string
```
Comment (sentientwaffle) [inline packages/coil-privacypass-sjcl/src/issuance.ts:41]: What's going on with these types?

Comment (sublimator) [reply packages/coil-privacypass-sjcl/src/issuance.ts:41]: A "TODO" WIP
Question is if it break things?
Please feel free to push a fix!

Comment (sublimator) [reply packages/coil-privacypass-sjcl/src/issuance.ts:41]: Will fix later, thanks!

**Comment by sublimator** [comment]
Thanks!

Will address when non-mobile!

**Comment by sublimator** [comment]
@wilsonianb fixed with https://github.com/coilhq/web-monetization-projects/pull/623/files

**Pull request merged**